### PR TITLE
Added program for Swapping Nodes in Linked List- Python

### PR DESCRIPTION
**Pull request by Pranav016**
### Have you read the [Contributing Guidelines on Pull Requests](https://github.com/TesseractCoding/NeoAlgo/blob/master/CONTRIBUTING.md#pull-requests)?

Yes

### Description

Added program for swapping nodes in a Linked List in Python. All **Pep8** rules have been followed and the code has been formatted. Doc strings for functions have been added, also inline and multi line comments have been added to explain the code.

### Checklist

- [x] I've read the contribution guidelines.
- [x] I've checked the issue list before deciding what to submit.
- [x] I've edited the `README.md` and link to my code.

## Related Issues or Pull Requests

#1572

### CODE REVIEW

**Comment by Pranav016** [comment]
Can any one of you please review this? @iamrajiv @atarax665 @shraddhavp @HarshCasper 
Thanks :)

**Review by iamrajiv — CHANGES_REQUESTED** [review]
@Pranav016 Format your code according to PEP 8 rules. You can refer to this http://pep8online.com/.

**Comment by Pranav016** [comment]
> @Pranav016 Format your code according to PEP 8 rules. You can refer to this http://pep8online.com/.

I already did, before making this PR. It is giving no errors 😄

File: Python/ds/SwapNodes-LinkedList.py [hunk 1]
```diff
@@ -0,0 +1,156 @@
...
+
+
+if __name__ == "__main__":
+    head = inputLL()
+    print("Enter index of nodes to be swapped-")
+    i, j = map(int, input().split())
+    head = swapNodes(head, i, j)
+    printLL(head)
+
+'''
```
Comment (atarax665) [inline Python/ds/SwapNodes-LinkedList.py:149]: Add space and time complexity of the program. Sample Input/Output should be exactly what we will get to see when we run the program, update it accordingly.

**Review by atarax665 — CHANGES_REQUESTED** [review]

File: Python/ds/SwapNodes-LinkedList.py (hunk 1 above)
Comment (Pranav016) [reply Python/ds/SwapNodes-LinkedList.py:149]: Updated 😄

**Comment by Pranav016** [comment]
> @Pranav016 Format your code according to PEP 8 rules. You can refer to this http://pep8online.com/.

Can you also suggest any VS code extension that helps to format the code like Pep8 ? That would be of great help! @iamrajiv

**Pull request merged**